packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx

```diff
@@ -117,7 +130,7 @@ const discoveryFailedMessage = (discovery?: Discovery) => {
 
         return value.concat(
             <div key={account.symbol}>
-                {n.name}: {account.error}
+                {n.name}: {getAccountError({ accountError: account.error })}
```

Comment:
nit: I think it can just accept param
```suggestion
                {n.name}: {getAccountError(account.error )}
```

```
const getAccountError = (accountError: string) => {
```